### ISSUE TITLE
Unable to set a page relation during Page Create Request

### ISSUE THREAD

**Issue by baxi87** [chojnac/NotionSwift#24]
Hi team!

Am creating a new page in a database, that includes a relation field ("User"). But am receiving this validation error message back from the API:

**failure(NotionSwift.NotionClientError.apiError(status: 400, code: "validation_error", message: "body failed validation: body.properties.User.relation[0] should be an object, instead was `\"a8073fb6-1fad-4aa8-93fc-31474eb77529\"`."))**

My original page request properties are set out as follows:

```
properties: [
                "Name": .init(
                    type: .title([
                        .init(string: "A new session was started")
                    ])
                ),
                "identifier": .init(
                    type: .richText([
                        .init(string: "\(self.user.id?.uuidString ?? "Unknown user")")
                    ])
                ),
                "User": .init(type: .relation([.init("a8073fb61fad4aa893fc31474eb77529")])),
                "Tags": .init(
                    type: .multiSelect([
                        .init(id: nil, name: "Hello", color: nil)
                    ])
                )
            ],

```

The original request output for the relation field shows as follows:
"User": NotionSwift.WritePageProperty(type: NotionSwift.PagePropertyType.relation([ID<Page>:a8073fb6-1fad-4aa8-93fc-31474eb77529]))

I've spent hours trying to resolve it, if the error is on my side, or you need more info to investigate. Do let me know.

**Comment by chojnac** [comment]
let me check

**Comment by chojnac** [comment]
It should be fixed in the latest release (0.7.1). Thanks for reporting the issue.
